src/components/OnboardingModal.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <ModalDefault
+    class="about-dex-modal"
+    close
+    @close="resolve"
+  >
+    <div class="wrapper">
+      <h1 class="info-headline">
+        {{ $t('onboardingModal.headline') }}
+      </h1>
+      <p class="info-text">
+        {{ $t('onboardingModal.body') }}
+      </p>
+      <p class="info-text">
+        <a
+          href="https://dex.superhero.com"
+          target="_blank"
+        >https://dex.superhero.com</a>
+      </p>
+      <ButtonDefault
+        class="confirm-button"
+        fill="primary"
+        @click.prevent="close"
+      >
+        {{ $t('onboardingModal.confirm') }}
+      </ButtonDefault>
+    </div>
+  </ModalDefault>
+</template>
+
+<script>
+import ButtonDefault from './ButtonDefault.vue';
+import ModalDefault from './ModalDefault.vue';
+
+export default {
+  components: {
+    ButtonDefault,
+    ModalDefault,
+  },
+  props: {
+    resolve: { type: Function, required: true },
+    reject: { type: Function, required: true },
+  },
+  methods: {
+    close() {
+      this.resolve();
+    },
+  },
```

Comment:
```suggestion
```
Can we just resolve it on click and not wrap it with method?

Reply:
fixed :)